content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
+description: An anaylsis of how cloud resources are utilized in ambient and sidecar service mesh architectures.
+publishdate: 2023-07-14
+attribution: "Greg Hanson - Solo.io"
+keywords: [istio,ambient,waypoint,ztunnel]
+---
+
+Last year the Istio community announced [ambient mesh](/blog/2022/introducing-ambient-mesh/), a new data plane mode which replaces sidecar proxies with per-namespace proxies.
+
+[Sidecars](/docs/reference/glossary/#sidecar) have been a staple of Istio’s architecture since day one, and are the way that most of Istio's features are provided today.  However, sidecars require the injection of an additional container to each Kubernetes `Pod` resource, and each sidecar container requires resources be allocated to it.
+
+Ambient mesh was released in [alpha](/docs/releases/feature-stages/#feature-phase-definitions) in [Istio 1.18](/news/releases/1.18.x/announcing-1.18/). This makes it easy to install, test, and to preview what it will mean for you when it's production-ready.
+
+Aside from the many other benefits of ambient mesh, the simple fact you no longer have to run as many proxies means that you can expect resource (and thus cost) savings. As you will see, these can be very substantial.
+
+## Check Out the Savings
+
+Our test scenario deploys four instances of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo), each to their own namespace, with each underlying microservice scaled to 3 replicas. There will be a total of 48 application services and 144 applications pods in this setup. Test scenarios utilize the provided [loadgenerator](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/loadgenerator) microservice running in each of the namespaces to generate traffic during the observed window. The tests deployed Istio in several scenarios and resource usage and allocation are compared across the runs. The scenarios are:
+
+- Istio with sidecar
+- Ambient with L4 ztunnel only
+- Ambient with L4 ztunnel and L7 waypoint proxies (one per namespace, scaled to 3 replicas)
+
+There are significant savings across the board for all ambient scenarios when compared to sidecars – up to 99% savings with only L4 and 89% with L4+L7.
+
+{{< image width="100%"
+    link="savings.png"
```

Comment:
Is it possible to convert this image into an editable table?

Reply:
yep... possible